openstack/er/v3/propagations/requests.go

```diff
@@ -0,0 +1,100 @@
+package propagations
+
+import (
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/pagination"
+)
+
+// ModifyOpts is the structure used to modify the propagation under a specified route table.
+type ModifyOpts struct {
```

Comment:
```suggestion
type PropagationOpts struct {
```

Reply:
CreateOpts and DeleteOpts are used to keep names consistent.